frb/galaxies/eazy.py

```diff
@@ -184,10 +190,20 @@ def eazy_input_files(photom, input_dir, name, out_dir, prior_filter=None,
     # Test combo
     assert combo in _acceptable_combos, "Allowed values of 'combo' are {}".format(_acceptable_combos)
 
+    # Test id col
+    if id_col is None:
+        if 'id' in photom.colnames:
+            id_col = "id"
```

Comment:
you can pull `id_col = "id"` out of the inner if/else block
as you set it as such in both cases.

Reply:
Done.